src/settings/FixedDueDateSchedule/components/EditSections/components/ScheduleCard/ScheduleCard.js

```diff
@@ -75,12 +80,12 @@ class ScheduleCard extends React.Component {
                   sm={4}
                 >
                   <Field
-                    backendDateStandard="YYYY-MM-DD"
+                    dateFormat="YYYY/MM/DD"
```

Comment:
Could you please move "YYYY/MM/DD" to constant and reuse it across all file?

Reply:
Done